inbound/supplierincomingmq/incoming_mq.py

```diff
@@ -0,0 +1,5 @@
+
+class SupplierIncomingMQ:
```

Comment:
@Broom94, please rename the package 'supplierincomingmq' into 'supplier_incoming_mq'